### PR TITLE
Document http_headers_getsetcookie compatibility flag

### PR DESCRIPTION
**Pull request by irvinebroque**
refs https://github.com/cloudflare/workerd/pull/321

cc @jasnell @GregBrimble from internal chat

https://community.cloudflare.com/t/2023-2-27-workers-runtime-release-notes/478334

### CODE REVIEW

**Comment by GregBrimble** [comment]
Related: #10586

File: content/workers/_partials/_platform-compatibility-dates/http_headers_getsetcookie.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  list: never
+
+name: "Get set cookie"
+sort_date: "2023-03-01"
+enable_date: "2023-03-01"
+enable_flag: "http_headers_getsetcookie"
+disable_flag: "http_headers_getsetcookie"
+---
+
+Adds the standard [`getSetCookie()`](https://developer.mozilla.org/en-US/docs/Web/API/Headers/getSetCookie) method to the [headers API](https://developer.mozilla.org/en-US/docs/Web/API/Headers) in Workers.
```
Comment (GregBrimble) [inline content/workers/_partials/_platform-compatibility-dates/http_headers_getsetcookie.md]:
```suggestion
Adds the [`getSetCookie()`](https://developer.mozilla.org/en-US/docs/Web/API/Headers/getSetCookie) method to the [Headers](https://developer.mozilla.org/en-US/docs/Web/API/Headers) API in Workers.
```

File: content/workers/_partials/_platform-compatibility-dates/http_headers_getsetcookie.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+_build:
+  publishResources: false
+  render: never
+  list: never
+
+name: "Get set cookie"
+sort_date: "2023-03-01"
+enable_date: "2023-03-01"
+enable_flag: "http_headers_getsetcookie"
+disable_flag: "http_headers_getsetcookie"
```
Comment (GregBrimble) [inline content/workers/_partials/_platform-compatibility-dates/http_headers_getsetcookie.md]:
```suggestion
disable_flag: "no_http_headers_getsetcookie"
```

File: content/workers/_partials/_platform-compatibility-dates/http_headers_getsetcookie.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+_build:
+  publishResources: false
+  render: never
+  list: never
+
+name: "Get set cookie"
```
Comment (jclee) [inline content/workers/_partials/_platform-compatibility-dates/http_headers_getsetcookie.md]:
```suggestion
name: "`Headers` supports `getSetCookie()`"
```

File: content/workers/_partials/_platform-compatibility-dates/http_headers_getsetcookie.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+enable_date: "2023-03-01"
+enable_flag: "http_headers_getsetcookie"
+disable_flag: "http_headers_getsetcookie"
+---
+
+Adds the standard [`getSetCookie()`](https://developer.mozilla.org/en-US/docs/Web/API/Headers/getSetCookie) method to the [headers API](https://developer.mozilla.org/en-US/docs/Web/API/Headers) in Workers.
+
+```js
+const response = await fetch("https://example.com");
+response.headers.getSetCookie();
```
Comment (jclee) [inline content/workers/_partials/_platform-compatibility-dates/http_headers_getsetcookie.md]:
```suggestion
let cookieValues = response.headers.getSetCookie();
```

**Comment by jclee** [comment]
Also, not sure if the existing documentation of the special treatment of cookie headers in Workers might also need updating...  https://developers.cloudflare.com/workers/runtime-apis/headers/#differences

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**